src/frontend/tests/extended/features/notifications.spec.ts

```diff
@@ -21,8 +21,10 @@ test(
       .getByTestId("inputsText Input")
       .hover()
       .then(async () => {
-        await page.getByTestId("add-component-button-text-input").click();
-        await page.getByTestId("button_run_text input").click();
+        await page.getByTestId("add-component-button-chat-input").click();
+        await page.getByTestId(/rf__node-ChatInput-/).click();
+        await page.keyboard.press('Control+.');
+        await page.getByTestId("button_run_chat input").click();
```

Comment:
Using the Text Input component is less dependent on other features and serves the same purpose
```suggestion
        await page.getByTestId("add-component-button-text-input").click();
        await page.getByTestId("button_run_text input").click();
```